src/UserSpaceInstrumentation/InjectLibraryInTracee.cpp

```diff
@@ -62,8 +63,9 @@ ErrorMessageOr<uint64_t> FindFunctionAddressWithFallback(pid_t pid, std::string_
 [[nodiscard]] ErrorMessageOr<void*> DlopenInTracee(pid_t pid, std::filesystem::path path,
                                                    uint32_t flag) {
   // Make sure file exists.
-  if (!std::filesystem::exists(path)) {
-    return ErrorMessage(absl::StrFormat("File not found (\"%s\")", path));
+  auto result_file_exists_or_error_or_error = orbit_base::FileExists(path);
```

Comment:
```suggestion
  auto result_file_exists_or_error = orbit_base::FileExists(path);
```
I guess this is a build error.

Reply:
Just `file_exists_or_error` like `code_address_or_error` below?

Reply:
done